src/main/kotlin/org/jetbrains/plugins/jumpToLine/JumpToLineArrowGutter.kt

```diff
@@ -126,10 +127,16 @@ internal class JumpToLineArrowGutter(
         private val WaitCursor = Cursor(Cursor.WAIT_CURSOR)
         private val smartStepInto =
                 EditorColorsManager.getInstance().globalScheme.getAttributes(DebuggerColors.SMART_STEP_INTO_TARGET)
-        private val greenColor = if (EditorColorsManager.getInstance().isDarkEditor) Color(30,40, 0) else Color(199,225, 94)
-        private val yellowColor = if (EditorColorsManager.getInstance().isDarkEditor) Color(50,40, 0) else Color(248,253, 185)
+        private val greenColor = JBColor.namedColor(
+            "JumpToLine.dropLine.safe",
+            JBColor(Color(199, 225, 94), Color(30, 40, 0))
+        )
+        private val yellowColor = JBColor.namedColor(
+            "JumpToLine.dropLine.unsafe",
+            JBColor(Color(248, 253, 185), Color(50, 40, 0))
+        )
         private val safeLineAttribute = smartStepInto.clone().also { it.backgroundColor = greenColor }
-        private val unsageLineAttribute = smartStepInto.clone().also { it.backgroundColor = yellowColor }
+        private val unsafeLineAttribute = smartStepInto.clone().also { it.backgroundColor = yellowColor }
```

Comment:
This was unrelated, was it supposed to be `unsafe`? I saw `LineSafetyStatus.NotSafe` and figured I'd change it.